docs/source/14_api_docs/kedro.extras.datasets.rst

```diff
@@ -13,12 +13,15 @@ Data Sets
     :template: autosummary/class.rst
 
     kedro.extras.datasets.api.APIDataSet
+    kedro.extras.datasets.avro.AVRODataSet
```

Comment:
Avro shouldn't be all capitalized, as it's not an abbreviation (here or anywhere after).

```suggestion
    kedro.extras.datasets.avro.AvroDataSet
```